tensorflow/lite/micro/tools/make/targets/cortex_m_corstone_300_makefile.inc

```diff
@@ -197,9 +194,13 @@ INCLUDES += \
   -I$(CMSIS_PATH)/Device/ARM/$(ARM_CPU)/Include \
   -I$(CMSIS_PATH)/CMSIS/Core/Include
 
-# TODO(#47070): Examine why some tests fail here.
+# TODO(#274): Examine why some tests fail here.
 EXCLUDED_TESTS := \
```

Comment:
I am a bit concerned of removing all these tests.  How much benefit do it gain by removing -O2?  I am a bit leaning on being safe since greedy_memory_planner are one important piece of the code and if it does not pass the test and the failure are genuine, debugging it on target would be a tremendous pain.  I totally agree on the need chasing down why this fails though and have create internal bugs to highlight this as well.

Reply:
One benefit of removing -O2 is that if not doing it more of these errors may accumulate in other unit tests, so having -O2 may hide these errors.

Reply:
ok. It is fair to expose issues rather than leaving it festing.